### ISSUE TITLE
Deploy through Bluemix pipeline issue

### ISSUE THREAD

**Issue by JFerguson576** [cfsworkload/engagement-advisor#2]
https://github.com/cfsworkload/watson-conversation 
When we deploy it to Bluemix it works. Then after a re-deploy it does not initiate with starter content and is unresponsive. We see it as being great for prototyping but we don't know why this happens and we can go no further. Any thoughts?

**Comment by dsayers** [comment]
Yes, I am aware of this issue. There is apparently an issue in the pipeline
build script that was not caught during testing. It may be a week or so
before I can try to fix it.

In the meantime, there is a work around.  Here is what someone else who
bumped into this issue did....

I got passed the problems we were seeing today.

The Environment Variable for DIALOG_ID kept getting a double-quote added to
the end.  Once I removed the double-quote, everything worked fine.  When I
look at the pipeline.yml file, I'm guessing if we change the lines that
sets the DIALOG_ID substring (e.g. DIALOG_ID=${DIALOG_ID:1:${#DIALOG_ID}-4
}) to remove an extra character; it may fix the problem.

I noticed updating the pipeline.yml file wasn't being used in the build
process; so I updated the "configure and deploy" configuration directly to
make the change.  It works as expected now.

**Comment by JFerguson576** [comment]
Thanks for looking into this. I look forward to redeploying it when you have made the changes this week. Regards, John
